keps/sig-storage/1979-object-storage-support/kep.yaml

```diff
@@ -2,9 +2,7 @@ title: Object Storage Support
 kep-number: 1979
```

Comment:
The images are still referring to BARs and BRs

Reply:
i've deleted the old images